### PR TITLE
Extend backwards-compatibility fixes

### PR DESCRIPTION
**Pull request by freethan**
WIP: Handles PR #93 and other multiple similar cases

### CODE REVIEW

File: Klarna.Rest/Klarna.Rest.Core.Examples/CustomerTokenApi.cs [hunk 1]
```diff
@@ -49,16 +48,16 @@ public void CreateOrderOnCustomerToken()
 
             var klarna = new Klarna(username, password, KlarnaEnvironment.TestingEurope);
             var customerToken = "234534...345345....345";
-            var orderData = new CustomerTokenOrder
+            var orderData = new Model.CustomerToken.CustomerTokenOrder//CustomerTokenOrder
```
Comment (alexions) [inline Klarna.Rest/Klarna.Rest.Core.Examples/CustomerTokenApi.cs]: Forgotten comment?

File: Klarna.Rest/Klarna.Rest.Core/Model/Enum/HostedPaymentPageStatus.cs [hunk 2]
```diff
@@ -5,11 +5,13 @@
...
     public enum HostedPaymentPageStatus
     {
-        WAITING, 
+        WAITING,
+        BACK,
         IN_PROGRESS,
         COMPLETED,
         CANCELLED,
         FAILED,
-        LEGACY
```
Comment (alexions) [inline Klarna.Rest/Klarna.Rest.Core/Model/Enum/HostedPaymentPageStatus.cs:13]: LEGACY is legacy? Removing this field will break the backward comaptibility

File: Klarna.Rest/Klarna.Rest.Core/Model/Enum/ShippingMethod.cs [hunk 3]
```diff
@@ -5,12 +5,14 @@ namespace Klarna.Rest.Core.Model.Enum
     /// </summary>
     public enum ShippingMethod
     {
-        PickupStore,
```
Comment (alexions) [inline Klarna.Rest/Klarna.Rest.Core/Model/Enum/ShippingMethod.cs:8]: This change will brake the backward compatibility

**Review by alexions — CHANGES_REQUESTED** [review]

File: Klarna.Rest/Klarna.Rest.Core/Model/Enum/HostedPaymentPageStatus.cs (hunk 2 above)
Comment (amirrsyd) [reply Klarna.Rest/Klarna.Rest.Core/Model/Enum/HostedPaymentPageStatus.cs]: Fixed

File: Klarna.Rest/Klarna.Rest.Core/Model/Enum/ShippingMethod.cs (hunk 3 above)
Comment (amirrsyd) [reply Klarna.Rest/Klarna.Rest.Core/Model/Enum/ShippingMethod.cs:8]: Added back enum

File: Klarna.Rest/Klarna.Rest.Core/Model/Enum/HostedPaymentPageStatus.cs (hunk 2 above)
Comment (amirrsyd) [reply Klarna.Rest/Klarna.Rest.Core/Model/Enum/HostedPaymentPageStatus.cs:13]: Added back enum

**Review by alexions — APPROVED** [review]

**Review by amirrsyd — APPROVED** [review]

**Pull request merged**